mlflow/sklearn/utils.py

```diff
@@ -328,14 +333,18 @@ def plot_confusion_matrix(*args, **kwargs):
         ),
     ]
 
+    is_plot_function_deprecated = Version(sklearn.__version__) >= Version("1.0")
```

Comment:
Can we move this function at line 297 and use it `plot_confusion_matrix` as well?